src/debug/utils.ts

```diff
@@ -76,10 +77,18 @@ export class PackageMap {
 					this.map[name] = uriToFilePath(rest);
 				else
 					this.map[name] = path.join(path.dirname(file), rest);
+
+				// If we map to "lib/" then this must be the local package so we can stash the name.
+				if (rest == "lib/")
```

Comment:
Is this sound? It seems logical to me that the current package always maps to `lib/` (and it always seems to show up at the end of the `.packages` file) but I don't know if that's solid or the best way (I can't think of any reason this wouldn't all be solid, but, noob etc.!)